code/extractors/PDFTextExtractor.php

```diff
@@ -42,6 +74,24 @@ function getContent($path) {
 		}
 		return implode('', $content);
 	}
-}
 
-?>
\ No newline at end of file
+	/**
+	 * Removes utf-8 ligatures.
+	 *
+	 * @link http://en.wikipedia.org/wiki/Typographic_ligature#Computer_typesetting
+	 *
+	 * @param string $input
+	 * @return string
+	 */
+	protected function cleanupLigatures($input) {
```

Comment:
While writing test cases. :) travis didn't have xpdf setup for testing so it was never covered.